lib/view_model/students.dart

```diff
@@ -66,7 +66,7 @@ class StudentsLogic extends ChangeNotifier {
     for (int i = 0; i < studentsDocIds.length; i++) {
       DocumentSnapshot<Object?> snap =
           await _collectionRef.doc(studentsDocIds[i]).get();
-      getTheStudent(snap);
+      _specificStudents.add(getTheStudent(snap));
```

Comment:
The variable name _specificStudents does not convey the use of the variable. A better name is needed.

Reply:
solved